examples/service-clients/odsp-client/shared-tree-demo/src/index.tsx

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import { ITree } from "@fluid-experimental/tree2";
+import { loadFluidData, containerSchema } from "./fluid";
+import { treeConfiguration, Letter } from "./schema";
+import { ReactApp } from "./reactApp";
+
+async function main() {
+	// create the root element for React
+	const app = document.createElement("div");
+	app.id = "app";
+	document.body.appendChild(app);
+
+	// Get the root item Id from the URL
+	// If there is no item Id, then the app will make
+	// a new container.
+	let itemId = location.hash.substring(1);
```

Comment:
Recommend computing the createNew state one time here and then using it below rather than repeatedly checking the length.

```suggestion
	// Get the root item Id from the URL
	let itemId = location.hash.substring(1);
	// If there is no item Id, then the app will make
	// a new container.
	const createNew = itemId.length === 0;
```